### PR TITLE
perf: Improve validation for usernameAvailable

### LINKED ISSUE #365: Improve validation for usernameAvailable query on client side
**Describe the bug**
We are receiving a lot of invalid requests to the usernameAvailable query which happens on the username screen.  We need to improve the client side validation so that these requests don't make it to the server.  I have created a [honeycomb query](https://ui.honeycomb.io/galoy/datasets/galoy-bbw/result/uyZJcn5MT1Y) to show the type of usernames which are being rejected server side.

### PR DESCRIPTION
**Pull request by danwebb1**
Resolves #365 
- Adds "lnbc1" to the `hasNoRestictedStartCharacters` function to reduce calls to the server. 
- Adds some clarity to the forbiddenStart message letting the user know the username cannot be a bitcoin address or lightning invoice
- Adds test for "lnbc1" 

*Note - I did my best to translate the forbidden start error message which may benefit from being proofed by a native Spanish speaker. The added language may not be even be necessary and can be removed also.

### CODE REVIEW

**Review by daviroo — CHANGES_REQUESTED** [review]
Hi @danwebb1.  Thanks for the contribution.  If you have the time it would be good to make the improvements I mentioned in my comments.  If you don't have time just let me know and I can take over the PR.

**Comment by danwebb1** [comment]
> Hi @danwebb1. Thanks for the contribution. If you have the time it would be good to make the improvements I mentioned in my comments. If you don't have time just let me know and I can take over the PR.

I will make the updates/improvements this evening!

File: app/i18n/en.json [hunk 1]
```diff
@@ -598,7 +598,7 @@
     "available": "✅  %{input} is available",
     "confirmSubtext": "The username is permanent and can not be changed later",
     "confirmTitle": "Set %{input} as your username?",
-    "forbiddenStart": "Cannot start with bc1, 1, or 3",
+    "forbiddenStart": "Cannot start with lnbc1, bc1, 1, or 3 and cannot be a Bitcoin address or Lightning invoice",
```
[the remark this answers is not in the document]
Comment (nicolasburtey) [reply app/i18n/en.json:601]: Email addresses will need to be working for lightning addresses

[the remark this answers is not in the document]
Comment (daviroo) [reply app/i18n/en.json:601]: @danwebb1 what @nicolasburtey is referring to is upcoming work we will do in the app to allow users to input static identifiers as outlined in the [lnurl spec](https://github.com/fiatjaf/lnurl-rfc/blob/luds/16.md).  Currently we don't support this.  If you want to leave out the email validation in anticipation of this change then that is fine with me.  Also ok if you have already implemented it as we can just remove it when we implement the above change.

**Comment by danwebb1** [comment]
https://user-images.githubusercontent.com/17483038/150078703-6331477e-4c6e-4af2-978a-cb5027c23df7.mov

**Review by daviroo — APPROVED** [review]

**Pull request merged**